chimney/src/test/scala/io/scalaland/chimney/PBTransformationSpec.scala

```diff
@@ -178,9 +178,37 @@ class PBTransformationSpec extends ChimneySpec {
 
   group("transformer sealed traits generated from oneof") {
 
-    test("CustomerStatus (oneof sealed_value)") {
+    object HandleEmptyAutomatically {
+      // TODO: consider creating some protobuf integration module in the future
```

Comment:
Actually, we could do this already:
* rename `protos` to `chimney-protobufs`
* add this utility there to the `main` scope
* move these tests to `protos`
* change the order of dependency: `chimney-protos` depending on `chimney` and make it publishable